SU2_CFD/src/fluid/CFluidFlamelet.cpp

```diff
@@ -0,0 +1,195 @@
+
+#include "../include/fluid/CFluidFlamelet.hpp"
+#include "../include/numerics/CLookUpTable.hpp"
+
+CFluidFlamelet::CFluidFlamelet(CConfig *config, su2double value_pressure_operating) : CFluidModel() {
+
+#ifdef HAVE_MPI
+  MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+#endif
```

Comment:
avoid the ifdefs, all methods of SU2_MPI will compile fine for serial compilation.

Reply:
we still have to convert cfluidflamelet...